resources/nl-core/nl-core-Patient.xml

```diff
@@ -73,6 +73,14 @@
         <profile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-AddressInformation" />
       </type>
     </element>
+    <element id="Patient.contact">
+      <path value="Patient.contact" />
+      <mapping>
+        <identity value="ART-DECOR-hg-Envelop" />
+        <map value="hg-dataelement-1677" />
+        <comment value="Contactpersoon" />
+      </mapping>
+    </element>

```

Comment:
Ik ken de context niet, maar let even goed op het verschil tussen Patient.contact en RelatedPerson. De eerste gaat alleen over de "persoon waarmee contact gezocht moet worden als er iets is met de patient".